app/services/publisher_wallet_getter.rb

```diff
@@ -11,22 +11,31 @@ def initialize(publisher:)
   def perform
     return perform_offline if Rails.application.secrets[:api_eyeshade_offline]
 
-    wallet_response = connection.get do |request|
-      request.headers["Authorization"] = api_authorization_header
-      request.url("/v1/owners/#{URI.escape(publisher.owner_identifier)}/wallet")
-    end
+    # Eyeshade only creates an account for an owner when they connect to Uphold.
+    # Until then, the request to get the wallet information for the owner will 404.
+    # In that case we use an empty wallet, but still use balances from the balance getter.
+    begin
+      wallet_response = connection.get do |request|
+        request.headers["Authorization"] = api_authorization_header
+        request.url("/v1/owners/#{URI.escape(publisher.owner_identifier)}/wallet")
+      end
 
-    wallet_hash = JSON.parse(wallet_response.body)
+      wallet_hash = JSON.parse(wallet_response.body)
+    rescue Faraday::ResourceNotFound
+      wallet_hash = {}
+    end
```

Comment:
Instead of making the request and rescuing upon a 404, I think we could also just set `wallet_hash = {}` if the publisher is not uphold verified and skip the request all together.  Not sure what's better in this case.  We could also do both.

Reply:
I think skipping the request if not uphold verified sounds slightly better to me